db/migrations/20221008153616_create_blockchain_transaction_table.js

```diff
@@ -13,7 +13,7 @@ exports.up = async (knex) => {
       .notNullable()
       .defaultTo('pending')
     t.timestamp('created_at').defaultTo(knex.fn.now())
-    t.timestamp('udpated_at').defaultTo(knex.fn.now())
+    t.timestamp('updated_at').defaultTo(knex.fn.now())
```

Comment:
Since the previous migration file was run, we should create another migration to alter this field.

Reply:
not run in  prod.  i have alter the schema in dev db by hand (This is not good..)

Reply:
emm.. that is very bad,  i forget there not only dev db, but also local db,  i will fix it